linux_os/guide/system/accounts/accounts-session/accounts_users_home_files_ownership/oval/shared.xml

```diff
@@ -0,0 +1,52 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("All User Files and Directories In The Home Directory Must Have a Valid Owner") }}}
+    <criteria>
+      <criterion test_ref="test_accounts_users_home_files_ownership"
+                 comment="All User Files and Directories In The Home Directory Must Have a Valid Owner"/>
+    </criteria>
+  </definition>
+
+  <unix:password_object id="object_accounts_users_home_files_ownership_objects" version="1">
+    <unix:username datatype="string" operation="not equal">nobody</unix:username>
+    <filter action="include">state_file_ownership_home_directories_interactive_uids</filter>
```

Comment:
State `state_file_ownership_home_directories_interactive_uids` is not defined anywhere.